test/e2e_tests/capp_revision_e2e_test.go

```diff
@@ -0,0 +1,73 @@
+package e2e_tests
+
+import (
+	"context"
+	"fmt"
+	mock "github.com/dana-team/container-app-operator/test/e2e_tests/mocks"
+	"github.com/dana-team/container-app-operator/test/e2e_tests/testconsts"
+	utilst "github.com/dana-team/container-app-operator/test/e2e_tests/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"strconv"
+)
+
+const (
+	MoreThanNumRevision = 12
```

Comment:
Rename this to `moreThanRevisionsToKeep` and also define this inside the `It` so it can be controlled by the `revisionsToKeep` const instead of it being its own const